pkg/localmetrics/localmetrics.go

```diff
@@ -42,10 +42,11 @@ type MetricsCollector struct {
 	accountClaims                   *prometheus.GaugeVec
 	accountReuseAvailable           *prometheus.GaugeVec
 	accountPoolSize                 *prometheus.GaugeVec
-	accountReadyDuration            prometheus.Summary
-	accountClaimReadyDuration       *prometheus.SummaryVec
-	accountReuseCleanupDuration     prometheus.Summary
+	accountReadyDuration            prometheus.Histogram
+	accountClaimReadyDuration       *prometheus.HistogramVec
+	accountReuseCleanupDuration     prometheus.Histogram
```

Comment:
These are used for the number of ready accounts/claims/reused accounts no?

Reply:
I didn't change these, they're in Ben's commit underneath this one. See #398.

Reply:
I'm just going to pull this off of @yeya24's PR and whoever comes second will have to resolve a merge conflict.

Reply:
Done.

Reply:
@jharrington22  These are used to track the ready time for an account/account claim CR. Please see https://issues.redhat.com/browse/OSD-4031